config/config.go

```diff
@@ -88,7 +107,10 @@ func (c Config) Save(path string) (err error) {
 // 3.  The value of $XDG_CONFIG_PATH/func if the environment variable exists.
 // The path is created if it does not already exist.
 func Path() (path string) {
```

Comment:
Wouldn't be better name `ConfigDir()`, from the name it's not clear if it returns path to global config, or dir containing it.

Reply:
Making the name imply if it is a filename or a directory sounds good to me. `config.ConfigDir()` is a bit redundant.  How about `config.Dir()` and `config.Filename()`?

Reply:
It think that's good.